libexec/mass-testing/readobj-json

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import argparse
+import json
+import sys
+from subprocess import PIPE, run
+
+import yaml
+
+
+def readobj(filepath: str):
+    proc = run(
+        ["llvm-readobj", "--elf-output-style=JSON", "--sections", filepath],
+        stdout=PIPE,
+        check=True,
+        text=True,
+    )
+    data = yaml.safe_load(proc.stdout)
+
+    result = {}
+    if len(data) == 1:
```

Comment:
1. There has to be a better way
2. Are the ELF and PE outputs similar?

Reply:
1. Eh, no
2. Not quite, the `ELF` output is proper JSON, but the PE/Mach-O gets "broken" by `--elf-output-style=JSON` enough that it is valid YAML

Reply:
2. > PE/Mach-O gets "broken" by --elf-output-style=JSON enough that it is valid YAML

   What do you mean?